salt/metalk8s/metalk8s-ui/deployed.sls

```diff
@@ -31,3 +46,16 @@ Create metalk8s-ui service:
         type: NodePort
   require:
     - pkg: Install Python Kubernetes client
+
+Create ui-api-configmap ConfigMap:
+  kubernetes.configmap_present:
+    - name: ui-api-configmap
+    - namespace: kube-system
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }} 
+    - data:
+        config.json: |
+          {"ip": "{{ control_plane_ip }}", "port": "6443"}
+  require:
+    - pkg: Install Python Kubernetes client
+    - file: Create metalk8s-ui directories
```

Comment:
What for?

Reply:
we need the directory ```ui``` to mount volume inside

Reply:
No, not on the host.

Reply:
my bad. your are right :) I mixed up host and container :(